ruby/red-arrow/test/values/test-map-array.rb

```diff
@@ -261,9 +261,10 @@ def test_time64_micro
 
   def test_time64_nano
     unit = Arrow::TimeUnit::NANO
+    # 00:10:00.123456789
+    key1_value = Arrow::Time.new(unit, (60 * 10) * 1_000_000_000 + 123_456_789)
     values = [
-      # 00:10:00.123456789
-      {"key1" => Arrow::Time.new(unit, (60 * 10) * 1_000_000_000 + 123_456_789), "key2" => nil},
+      {"key1" => key1_value, "key2" => nil},
```

Comment:
```suggestion
      {"key1" => value, "key2" => nil},
```

Reply:
Thanks. I thought `value` was already used in line 266. But It was `valueS`. Commit your suggestions.